database/user.go

```diff
@@ -334,8 +330,18 @@ func (db *DB) UserCreate(ctx context.Context, emailAddr, pass, sub string, tier
 		EmailConfirmationToken:           emailConfToken,
 		EmailConfirmationTokenExpiration: time.Now().UTC().Add(EmailConfirmationTokenTTL).Truncate(time.Millisecond),
 		PasswordHash:                     string(passHash),
+		RecoveryToken:                    "",
 		Sub:                              sub,
 		Tier:                             tier,
+		CreatedAt:                        time.Now().UTC(),
+		MigratedAt:                       time.Time{},
+		SubscribedUntil:                  time.Time{},
+		SubscriptionStatus:               "",
+		SubscriptionCancelAt:             time.Time{},
+		SubscriptionCancelAtPeriodEnd:    false,
+		StripeID:                         "",
+		QuotaExceeded:                    false,
+		PubKeys:                          make([]PubKey, 0),
```

Comment:
this removes the need for that type query right?

Reply:
We still have all pre-existing users who don't have any pubkeys.